sdk/metric/benchmark_test.go

```diff
@@ -179,8 +198,6 @@ func BenchmarkIterator_16(b *testing.B) {
 	benchmarkIterator(b, 16)
 }
 
-// Counters
```

Comment:
Sorry, there was a typo in my previous comment. I meant should it be skipped (not shipped) instead of being fully commented.
Fully commented test cases can be easy to forget. While a pending/skipped test case remains visible.

Reply:
Yeah I understood, but it could just be commented because of the lack of `global` dependency. I think it could be fixed after the global package is ready in `sdk/metric`.